doc/source/state-machine.rst

```diff
@@ -166,28 +166,14 @@ RE.state       Check if 'paused' or 'idle'.
 Automated Suspension
 ====================
 
-It can also be useful to interrupt execution automatically in response some
-condition (e.g., shutter closed, beam dumped, temperature exceed some limit).
+It can also be useful to interrupt execution automatically in response to some
+condition (e.g., shutter closed, beam dumped, temperature exceeded some limit).
 We use the word *suspension* to mean an unplanned pause initialized by some
 agent running the background. The agent (a "suspender") monitors some condition
 and, if it detects a problem, it suspends execution. When it detects that
 conditions have returned to normal, it gives the RunEngine permission to resume
 after some interval. This can operate unattended.
 
-.. ipython::
-    :verbatim:
-
-    In [1]: RE(scan([det], motor, -10, 10, 15), LiveTable([motor, det]))
-    +------------+-------------------+----------------+----------------+
-    |   seq_num  |             time  |         motor  |           det  |
-    +------------+-------------------+----------------+----------------+
-    |         1  |  16:46:08.953815  |          0.03  |        290.00  |
-    Suspending....To get prompt hit Ctrl-C to pause the scan
-    |         2  |  16:46:20.868445  |          0.09  |        279.00  |
-    |         3  |  16:46:29.077690  |          0.16  |        284.00  |
-    |         4  |  16:46:33.540643  |          0.23  |        278.00  |
-    +------------+-------------------+----------------+----------------+
-
```

Comment:
@danielballan, I removed this example since exactly the same example appear later.

Reply:
I actually prefer this duplication to stay. My intention was to provide a quick, easy-to-skim example to help people understand what we mean by "suspension", even if they don't dig into the details of the full example just yet.